src/main.rs

```diff
@@ -107,16 +107,14 @@ extern "C" fn sample_pending() {
 
     loop {
         ui::SingleMessage::new("Pending").show();
-        match comm.next_event::<Ins>() {
-            Event::Button(ButtonEvent::RightButtonRelease) => break,
-            _ => (),
+        if let Event::Button(ButtonEvent::RightButtonRelease) = comm.next_event::<Ins>() {
+            break;
         }
     }
     loop {
         ui::SingleMessage::new("Ledger review").show();
-        match comm.next_event::<Ins>() {
-            Event::Button(ButtonEvent::BothButtonsRelease) => break,
-            _ => (),
+        if let Event::Button(ButtonEvent::BothButtonsRelease) = comm.next_event::<Ins>() {
+            break;
         }
     }
```

Comment:
Shouldn't this part made mor similar than https://github.com/LedgerHQ/app-age/commit/a1ce54e56b11b2a6418608268590bde93c50202f ?

Reply:
Yep no problem. Simpler in app-age